OasisDEX/oasis-borrow
# Display values manage page V3L1

# [Display values manage page V3L1](https://app.shortcut.com/oazo-apps/story/7740/fe-manage-displaying-data-from-l1v3-in-the-views)
  
## Changes 👷‍♀️
  <Please add short list of changes>
- reads data from IPosition instead using latest version of lib to get oracle price
  
## How to test 🧪

- create a position
- load manage page
- ensure values correct
- ensure no regression on earn steth/eth V2 manage
